complete/src/main/java/com/example/restservice/controller/QueueController.java

```diff
@@ -1,10 +1,10 @@
 package com.example.restservice.controller;
 
-import com.example.restservice.model.CreateQueueRequest;
-import com.example.restservice.model.CreateQueueResponse;
-import com.example.restservice.model.MyQueuesResponse;
-import com.example.restservice.model.QueueDetailsResponse;
-import com.example.restservice.model.QueueStatusResponse;
+import com.example.restservice.controller.model.queue.CreateQueueRequest;
```

Comment:
We're putting our models inside our controller package?

Reply:
I don't know the right word to call this. This is not "model" it should be called something else. It's the request / response  body POJO. What do we call it?